README.md

```diff
@@ -1,14 +1,14 @@
 # pg-schema-diff
+Diffs Postgres database schemas and generates the SQL required to get your database schema from point A to B while 
+minimizing downtime and locks. This enables you to take your database and migrate it to any desired schema defined with plain DDL.
```

Comment:
"while minimizing downtime and locks" reads like a run on sentence because it's a third clause. Maybe "with minimal downtime & locks" helps a bit? Or break it into multiple sentences somehow.